source/aws_lambda/custom_resource/remove_s3_object/remove_s3_object.py

```diff
@@ -0,0 +1,48 @@
+# ######################################################################################################################
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.                                                  #
+#                                                                                                                      #
+#  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance      #
+#  with the License. You may obtain a copy of the License at                                                           #
+#                                                                                                                      #
+#   http://www.apache.org/licenses/LICENSE-2.0                                                                         #
+#                                                                                                                      #
+#  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed    #
+#  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for   #
+#  the specific language governing permissions and limitations under the License.                                      #
+# ######################################################################################################################
+"""
+This module is a custom lambda to remove empty file object place in the inbound bucket
+"""
+
+from aws_lambda_powertools import Logger
+from crhelper import CfnResource
+from aws_solutions.core.helpers import get_service_client
+
+logger = Logger(utc=True, service='transform-custom-lambda')
+helper = CfnResource(log_level="ERROR", boto_level="ERROR")
+
+
+def event_handler(event, context):
+    """
+    This is the Lambda custom resource entry point.
+    """
+    logger.info(event)
+    helper(event, context)
+
+
+@helper.create
```

Comment:
def is `on_create_or_update`, do we need `@helper.update` decorator here?

Reply:
We need it only on create. I'll rename the function. thanks